hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -328,10 +328,22 @@ def __init__(self, client, id, attributes):
     async def status(self, include_jobs=True):
         resp = await self._client._get(f'/api/v1alpha/batches/{self.id}')
         batch = await resp.json()
+
         if include_jobs:
-            resp = await self._client._get(f'/api/v1alpha/batches/{self.id}/jobs')
-            jobs = await resp.json()
+            jobs = []
+            last_job_id = None
+            while True:
+                params = {}
```

Comment:
Would it be better to have a different interface than having the jobs on the status? Then we could return an iterator. Although I'm not sure how that would work with the regular synchronous client.

This isn't something we need to do now for this PR if you think changing the interface is a good idea.

Reply:
Yes, 100%, I'm already planning to do that in another PR.  Yeah, the sync one should return an iterator, the async one an async iterator.